packages/rollup-plugin-minify-html-literals/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@lit/rollup-plugin-minify-html-literals",
+  "version": "1.0.0",
```

Comment:
Could this be change to version "0.0.0" initially (I believe this will be bumped by our changesets deploy process).

Also please add: `"private": true,` (since there is nothing in this package yet and we don't want to accidentally publish it).

Reply:
Thanks for the review Andrew. I have made the recommended changes. For the root package-lock, I included the result after `npm install` from within the monorepo root.

Reply:
Thank you - looks good to me!